charts/kcp/values.yaml

```diff
@@ -134,6 +139,36 @@ kcpFrontProxy:
   # - name: example-vw-serving-cert
   #   mountPath: /etc/example-vw-serving-cert
   extraFlags: []
+cache:
+  enabled: false
+  image: ghcr.io/kcp-dev/kcp
+  tag: "main"
```

Comment:
this should match what we do for other components, `tag` should be empty and default to the appVersion of the chart.

Reply:
It's not even respected in `cache-deployment.yaml`:

```
image: {{ .Values.cache.image }}:{{- include "kcp.version" . }}
```